jobs/tasks/telegraf.cue

```diff
@@ -40,7 +39,7 @@ import (
 		[inputs.prometheus]
 		metric_version = 1
 
-		urls = [ "http://{{ env "NOMAD_ADDR_\(#prometheusPort)" }}" ]
+                urls = [ "http://{{ env "NOMAD_ADDR_rest" }}/prometheus" ]
```

Comment:
This needs to be indented with tabs like the others.

Reply:
Yes, saw the error, handled the whitespace correctly now. Thanks!